worksheet_B/Mandelbrot set working.cpp

```diff
@@ -0,0 +1,97 @@
+// ConsoleApplication3.cpp : Defines the entry point for the console application.
+//
+
+#include "stdafx.h"
+
+float ImageWidth = 800.0;
+float ImageHeight = 800.0;
+int Iteration = 0;
+int maxIteration = 200;
+int Red = 0;
+int Green = 0;
+int Blue = 0;
+int minValue = 0;
+int maxValue = 255;
+
+int main()
+{
+	// Initialise the image
+	CImg<unsigned char> image(ImageHeight, ImageWidth, 1, 3, 0);
+	// Display the image
+	CImgDisplay display(image, "Mandelbrot Set");
+	// Minimum and maximum coordinates for the fractal
+	const double minX = -2, maxX = 1, minY = -1.5, maxY = 1.5;
+	// Generate the image
+	for (int pixelY = 0; pixelY < image.height(); pixelY++)
+	{
+		//Map the y coordinate into the range minY to maxY0
+		double y0 = ((pixelY / ImageWidth) * (maxY - minY) + minY);

```

Comment:
Shouldn't you be using `ImageHeight` here? (In this case they happen to be equal so it doesn't matter, but still...)